Tasks/ServiceFabricUpdateAppVersions/Compare-Xml.psm1

```diff
@@ -0,0 +1,25 @@
+﻿function Compare-Xml
+{
+    [CmdletBinding()]
+    [OutputType([bool])]
+    Param
+    (
+        [Parameter(Mandatory=$true)]
+        [System.Xml.XmlNode]
+        $Item1,
+
+        [Parameter(Mandatory=$true)]
+        [System.Xml.XmlNode]
+        $Item2
+    )
+
+    Add-Type -AssemblyName System.Xml.Linq
+
+    $xmlReader1 = [System.Xml.XmlNodeReader] $Item1
+    $xmlReader2 = [System.Xml.XmlNodeReader] $Item2
+
+    $xDoc1 = [System.Xml.Linq.XDocument]::Load($xmlReader1)
+    $xDoc2 = [System.Xml.Linq.XDocument]::Load($xmlReader2)
+
+    [System.Xml.Linq.XNode]::DeepEquals($xDoc1, $xDoc2)
```

Comment:
I checked each call into [System.Xml.Linq](https://msdn.microsoft.com/en-us/library/system.xml.linq.xnode.deepequals(v=vs.90).aspx) and it exists back to .Net 3.5. (System.Xml.XmlNodeReader also exists in 3.5)